pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/windowing/WindowContextImpl.java

```diff
@@ -80,6 +87,11 @@ public String getOutputTopic() {
         return this.context.getOutputTopic();
     }
 
+    @Override
+    public Record<?> getCurrentRecord() {
```

Comment:
I think it's best to keep the approach of defining a separate interface for WindowsContext. For example, in the semantics of Windows functions, which record does `getCurrentRecord` refer to? We cannot provide this method, right?

Reply:
changed to extend `BaseContext`, this is more reasonable